<launch>
  <rosparam file="$(find pr2_default_controllers)/gripper_effort_controller.yaml" command="load" ns="r_gripper_effort_controller" />
  <node pkg="pr2_mechanism_control" type="spawner.py" name="grip_ctrl"
        args="r_gripper_effort_controller" />  
  
  <node pkg="qualification" type="fingertip_qualification.py"
        name="tip_analysis" output="screen">
    <param name="grasp_force_initial" type="double" value="-25.0" />
    <param name="grasp_force_increment" type="double" value="-1.0" />
    <param name="grasp_increments" type="int" value="75" />
    <param name="x^0" type="double" value="701.63"/>
    <param name="x^1" type="double" value="-896.61" />
    <param name="x^2" type="double" value="0" />
    <param name="x^3" type="double" value="0" />
    <param name="tolerance_percent" type="double" value="10.0"/>
    <param name="fingertip_refresh_hz" type="double" value="25.0"/>
    <param name="num_sensors" type="int" value="22"/>
    <param name="num_sensors_outside" type="int" value="7"/>
  </node>
</launch>
